content/structure/PollOption.md

```diff
@@ -0,0 +1,76 @@
+**Project NewCaw** \
+Author: Frederick Schenk
+
+# Content/PollOption
+
+![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public struct PollOption
+```
+
+A small helper structure for storing options in a [`Poll`](../class/Poll.md).
+
+## Fields
+
+### `position`
+
+![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square)\
+
+```c#
+public int position
+```
+
+The position of the option in the Poll.
+
+> Probably unnecessary, for example Mastodon does not include it as well.
```

Comment:
We can probably just use this as a way to get the ordering correct at load time, and then just rely on the ordering of the array.